src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Combination/paginated_list_header.html.twig

```diff
@@ -63,8 +68,30 @@
         data-view-error-log="{{ 'View %error_count% error logs'|trans({}, 'Admin.Catalog.Feature') }}"
         data-view-error-title="{{ 'Error log'|trans({}, 'Admin.Catalog.Feature') }}"
       >
-        {{ 'Delete %combinations_number% combinations'|trans({'%s': ''}, 'Admin.Catalog.Feature') }}
+        {{ deleteLabel }}
       </button>
+      {% if isMultistoreActive %}
+        <button
+          id="combination-bulk-delete-btn-all-shops" class="dropdown-item bulk-action-btn bulk-delete-btn" type="button"
+          data-btn-label="{{ 'Delete %combinations_number% combinations for all shops'|trans({}, 'Admin.Catalog.Feature') }}"
+          data-form-url="{{ url('admin_products_combinations_bulk_delete', { 'productId': productId }) }}"
+          data-modal-message="{{ 'Are you sure you want to delete selected combinations?'|trans({}, 'Admin.Catalog.Notification') }}"
```

Comment:
```suggestion
          data-modal-message="{{ 'Are you sure you want to delete selected combinations for all shops?'|trans({}, 'Admin.Catalog.Notification') }}"
```

Reply:
Maybe not worth it to adapt all the labels with the `for all shops` but at least the confirmation message could avoid mistakes

Reply:
labels must change too, because there are 2 buttons for deletion now (one for single shop and one for all shops)

Reply:
Yeah I agree basically what I meant is `data-btn-label` and `data-modal-message` should both be adapted when we are in multi-shop, so that the button label AND the message in the modal indicate explicitly if the delete targets a single shop or all of them